api/src/pcapi/core/users/models.py

```diff
@@ -726,8 +727,10 @@ def has_partner_page(self) -> bool:
             .join(Venue, UserOfferer.offererId == Venue.managingOffererId)
             .where(
                 UserOfferer.userId == self.id,
+                Offerer.isActive.is_(True),
                 Venue.isPermanent.is_(True),
                 Venue.isVirtual.is_(False),
+                Venue.hasOffers == True,
```

Comment:
[question] est-ce que c'est parce que c'est une `property` et du coup on ne peut pas utiliser `is_` et faire:
```suggestion
                Venue.hasOffers.is_(True),
```

Reply:
C'est au niveau du typage que ça bloque : `error: overloaded function has no attribute "is_"` 
Cepedant je vois dans le code d'autres endroit où c'est utilisé ([ici](https://github.com/pass-culture/pass-culture-main/blob/master/api/tests/core/bookings/test_models.py#L266) par exemple) mais dans les tests, fichiers exclus de mypy (cf [ici](https://github.com/pass-culture/pass-culture-main/blob/master/api/pyproject.toml#L162C1-L167C4))